app/src/main/java/io/hobaskos/event/eventapp/ui/profile/ProfileActivity.java

```diff
@@ -26,9 +26,19 @@
 
     private static final String TAG = "ProfileActivity";
 
+    /**
+     *
```

Comment:
Når man kjører Log.i( "ProfileActivity", "string") for å feilsøke så kan det være hendig å ha det som en Tag.